### ISSUE TITLE
[BUG] Cannot Sign in with Apple ID on SideStore 

### ISSUE THREAD

**Issue by Renny242** [SideStore/SideStore#768]
### Describe the bug

I can’t sign into to my Apple ID it just says something like an error and I discovered when I was signing in:
![IMG_1153](https://github.com/user-attachments/assets/8ddbf871-befe-4597-90f2-7fe92c7ce198)

### Instructions to reproduce

Just fix the bug or issue with signing in with Apple ID or use something like google or something please, thank you

### What version of SideStore are you using?

I’m using SideStore 0.5.8

### Other info

I really want this fix and I have been dreaming about using this app to its full potential.

**Comment by mekabrine** [comment]
**cant login with apple account?**
> go to sidestore settings

> scroll to bottom

> tap "Anisette Servers"

> select the server "neoarz"
> -# https://neos-anisette-server.onrender.com

> tap "Back"

> attempt to log into your apple account again

**Comment by neoarz** [comment]
> **cant login with apple account?**
> 
> > go to sidestore settings
> 
> > scroll to bottom
> 
> > tap "Anisette Servers"
> 
> > select the server "neoarz"
> > -# https://neos-anisette-server.onrender.com
> 
> > tap "Back"
> 
> > attempt to log into your apple account again

aww lol thanks but lets stick to giving support with official servers for now

**Comment by nophramel** [comment]
> **cant login with apple account?**
> 
> > go to sidestore settings
> 
> > scroll to bottom
> 
> > tap "Anisette Servers"
> 
> > select the server "neoarz"
> > -# https://neos-anisette-server.onrender.com
> 
> > tap "Back"
> 
> > attempt to log into your apple account again

can't find the server neoarz

**Comment by nophramel** [comment]
> Copy and paste the link

tried it, but it seems it gets over written. I copy pasta your link and enter. then i use the back button.
I think i have a different problem since i still get the prompt to enter the 6-digit code

**Comment by tchlyah** [comment]
Hi, 

I'm on iOS 18.2, I tried the above method using iCloud.com, and changing anisette servers as well, but nothing works, I always have wrong verification code.

It's impossible to use this app. How can I fix that ?

Thank you

**Comment by Scvairy** [comment]
> **Dont get Verification code when signing into Sidestore?** _For ios versions below 18.1_
> 
> > Go to settings
> 
> > Tap on your name
> 
> > Sign in and security
> 
> > Two factor authentication
> 
> > Get verification code

I didn't find it on iOS 18.3 beta 2, but found it on macOS. The first code was somehow 5-digit long, but the next one had 6 and it worked just fine.

**Comment by mahee96** [comment]
yeah possibly anisette was down, if issue is still present open new ticket and link this issue
